sdk/src/error.rs

```diff
@@ -207,6 +210,12 @@ pub enum Error {
     #[error("must fetch remote manifests from url {0}")]
     RemoteManifestUrl(String),
 
+    #[error("failed to fetch the remote settings")]
```

Comment:
I don't want to release all these new errors before an error rework, but lets get them in and I'll work on that.